docs/user/tutorial/map/style.rst

```diff
@@ -135,7 +135,7 @@ Dependencies
 Please ensure your pom.xml includes the following::
 
   <properties>
-      <geotools.version>2.7-SNAPSHOT</geotools.version>
+      <geotools.version>14.0</geotools.version>
```

Comment:
@ianturton I'm wondering about the Release Version on master-branch. Is it because demo-code is tested against this version? Is it worth to think about  @RELEASE@ token or the Shinx replacemant |version| that could be replaced with the final release version during release-procedure (rename.xml) or during Sphinx-doc build?